docs/2_2_common_schema.adoc

```diff
@@ -1159,6 +1159,19 @@ It should be noted that changing a <<structuralParameter,`structural parameter`>
 
 It is not possible to use a variable of type `Clock` in arrays, since array elements do not have individual <<valueReference>> and the access to <<clock,`clocks`>> requires a unique <<valueReference>> information.
 
+A variable can have any number of `<Alias>` elements that define a variable alias.
+Each variable alias has a required attribute `name` whose value must be unique among all variables and variable aliases and an optional attribute `description`.
+Variable aliases of numeric variables can optionally be defined with the attribute `negated = true` which indicates that the variable's value must be multiplied with `-1` to retrieve the value of the variable alias.
```

Comment:
Do we really want to reintroduce negated alias variables?

Reply:
I am against negated variables and the corresponding attiribute.
This complicates the work of the importer. 
Why not have a separate variable with the negative value?

Reply:
The whole point of aliases (as I understand it) is to make tracing variables more efficient. We decided that aliases are just additional information and can be completely neglected by the importer.

Reply:
> The whole point of aliases (as I understand it) is to make tracing variables more efficient. We decided that aliases are just additional information and can be completely neglected by the importer.

But especially then I would get rid of the negated variables. 
(The reason to have them in FMI 2.0 to my understanding was to have e.g. two outputs referring to the same value reference, but with opposite sign, enabling the fulfillment of sign conventions, e.g. for the forces on both sides of a spring)

Reply:
The re-introduction of negated aliases also threw me for a loop: They were _removed_ in 2.0, so I think we should have a good reason to reintroduce them in 3.0. Given that the aliases are now completely ignorable add-on information, if we allow negated stuff, why not other linear (or non-linear for that matter) transformations (alternative units already kind of give you this ability partially through misuse)?

This seems to complexify stuff needlessly: Without negated aliases, this would at least allow the non-floating-point variables to use the same elements (and the FP stuff already has more specialties), thus reducing schema complexity as well.

Given that 2.0 did not support negated aliases, and I haven't seen any major complaints, why re-introduce them?

Reply:
Can we not mimic negated variables by just using display units?
For instance:
```
<UnitDefinitions>
  <Unit name="kg.m2">
    <BaseUnit kg="1" m="2"/>
    <DisplayUnit name="-kg.m2" factor="-1"/>
  </Unit>
</UnitDefinitions>
...
<Float32 name="J" valueReference="1073741824"
  description="Moment of load inertia" causality="output" variability="continuous"
  unit="kg.m2" start="1">
  <Alias name="-J" description="Negation example" displayUnit="-kg.m2"/>
</Float32>
```

Reply:
I find the `negated` attribute more elegant than the "unit solution" because it also covers variables that have no unit.
However, if this this is the only thing that's holding you back from approving this PR, I can remove it. We could still add it later.

Reply:
How would a `negated` option work with min/max attributes for a variable?